src/Pkg.jl

```diff
@@ -520,11 +520,14 @@ packages holding them back due to compatibility constraints, or due to compatibi
 As an example, a status output like:
 ```
 pkg> Pkg.status(; outdated=true)
```

Comment:
This should be a julia prompt then?

Reply:
Julia prompt is `julia-repl`?
Adding this does not highlight anything then.

Reply:
Now I see what you mean. I fixed it.